app/assets/javascripts/vendor/lib/google/maps.js.erb

```diff
@@ -1,419 +1,417 @@
-        var xmp = xminp + 0.5 * wp;     // midpoint of east
-        var wxp;                    // width to anti/prime meridian east
-        var wxm;                    // width to anti/prime meridian west
-        var wx;                               // total width of all areas
-        //wx = xmaxp - xminm;             // assume data in both hemispheres default across prime meridian
-        wx = wm + wp;             // assume data in both hemispheres default across prime meridian
-        if (wx > 180) {               // areas are > 180 degrees wide or cross prime meridian
-          if (wm == 0 && wp == 0) {   // this is the case of single longitude in each (both!) hemisphere
-            wx = 360 - wx;                              // so revert to < 180 degree width (smaller distance)
-            center_long = (xminm + xmaxp) / 2 - 180;   // left-hand expression is near antimerdian
-          }
-          else {
-            if (wm == 0 || wp == 0) {                // only one point in either hemisphere and multiples/areas in the opposite
-              if (wm == 0) {     // if single point in western hemisphere
-                wxp = 180 - xminp;    // xmaxp = xminp for single point
-                wxm = 180 + xmaxm;    // western half
-                wx = wxp + wxm;
-                if (wxm > wxp) {      // is western hemisphere part bigger?
-                  center_long = xmaxm - wx / 2 - 180;
-                  wx = wx - 0;     ////  DUMMY
-                }
-                else {      // eastern width is greater
-                  center_long = xminp + wx / 2;
-                  wx = wx - 0;     ////  DUMMY
-                }
-              }
-              else {        // wp = 0
-                wxp = 180 - xminp;    // xmaxp = xminp for single point
-                wxm = 180 + xmaxm;    // western half
-                wx = wxp + wxm;
-                if (wxm > wxp) {
-                  center_long = -0.5 * (wxp + wxm);
-                  center_long = (xminp + xmaxm) / 2 - 180;
-                  wx = wx - 0;     ////  DUMMY
-                }
-                else {
-                  center_long = 0.5 * (wxp + wxm);
-                  wx = wx - 0;     ////  DUMMY
-                }
-              }
+        var wm = 0.0;        // western hemisphere default area width
+        var wp = 0.0;        // eastern hemisphere default area width
+        if ((xminp == 180.0) && (xmaxp == 0.0)) {    //if no points, null out the range for this hemisphere
+          xminp = 0.0;                                // possible place to mark singleton point
+        }
+        if ((xmaxm == -180.0) && (xminm == 0.0)) {    //if no points, null out the range for this hemisphere
+          xmaxm = 0.0;                                // possible place to mark singleton point
+        }
+        if ((xminp == 0.0) && (xmaxp == 0.0) && (xmaxm == 0.0) && (xminm == 0.0)) {   // no data (kinda)
```

Comment:
This is related to the deferred back-integration of anti-meridian handling into the maps code.